### DISCUSSION TITLE
Push Notification Icon shown as a white circle

### DISCUSSION THREAD

**Discussion by IbkStorm** [capawesome-team/capacitor-firebase#489] [Q&A] [answered]
I added the notification icon and set the path in the AndroidManifest.xml but the icon won't show when receiving notifications.
By default the icon was in the drawable folder but I also tried moving it to the mipmap folder and it still didn't work.
I changed your code for my ionic react application but I also made a new ionic angular app and the problem is the same.
I mainly test inside android studio with simulated devices but I also tried it on a physically device and the outcome is the same.

**Comment by robingenz** [comment]
From the [documentation](https://capawesome.io/plugins/firebase/cloud-messaging/#push-notification-icon):

> the push icon should be white pixels on a transparent background. 

Does your icon meet these requirements?

**Reply by IbkStorm** [reply]
Well it should be but when I look closly it might have some gray parts on the edges but it is on a transparent background.
Will it not work if any pixel isn't white? I will try and set it to be completly white on transparent background.

**Reply by IbkStorm** [reply]
I changed the image to be strictly white pixels on transparent background but the notification icon is still only showen as a white circle.

**Reply by robingenz** [reply, marked as answer]
Maybe this section from my guide helps you: https://capawesome.io/blog/the-push-notifications-guide-for-capacitor/#configure-the-push-notification-icon

Here you can find a working example: https://github.com/robingenz/ionic-capacitor-firebase-messaging-demo

**Reply by IbkStorm** [reply]
The line <meta-data android ... was placed at the wrong position in AndroidManifest.xml
Now it works. Thank you for your help.
